.github/workflows/publish-to-maven.yml

```diff
@@ -27,17 +27,20 @@ jobs:
 
     # CI release command defaults to publishSigned
     # Sonatype release command defaults to sonaTypeBundleRelease
-      - name: Gradle publish
-        run: gradle clean publish
+    # Changing env names as documented here - https://vanniktech.github.io/gradle-maven-publish-plugin/central/#secrets
+      - name: Gradle Build and Publish to Maven central
+        run: |
+          ./gradlew publish
```

Comment:
Also, do want to publish for all check-ins? or do we want to bring back the condition removed here?
https://github.com/feathr-ai/feathr/pull/974

Reply:
This check makes sure it only gets triggered for a new tag

Reply:
tags: ["*"]